root/io/filemerger/CMakeLists.txt

```diff
@@ -1,3 +1,9 @@
+if("${LZ4_VERSION}" STREQUAL "")
+   set(lz4_ver 1.8.3)
```

Comment:
Why 1.8.3 (as opposed to any other random version)?  @oshadura outside of ROOT's CMake do we have access to the version of lz4 used/available in ROOT?

Reply:
1.8.3 is lz4 version in ROOT master.
To be fully exact one should parse lz4.h header as ROOT cmake does

Reply:
I will take a look, it is bug introduced by me :( sorry for this...